docker-compose.yml

```diff
@@ -374,3 +376,11 @@ services:
     image: ghcr.io/iden3/driver-did-iden3:v0.0.4
     ports:
       - "8152:8080"
+  
+  driver-did-earthid:
+    image: vibhi09/driver-did-earthid:latest
```

Comment:
Please use a stable version instead of "latest"

Reply:
Hi @peacekeeper , I've updated the repo with a stable version and updated port number kindly review my latest commit in the current pull request and let me know if any changes required.
Thank you!